api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimRepositoryTest.java

```diff
@@ -244,4 +250,21 @@ void shouldReturnNewClaimIds() {
         //Then
         assertThat(result).containsOnly(newClaim.getId());
     }
+
+    @Test
+    void shouldAuditClaimUpdate() {
+        //Given
+        Claim claim = aClaimWithClaimStatus(ACTIVE);
+        claimRepository.save(claim);
+        claim.setClaimStatus(PENDING);
+        claimRepository.save(claim);
+
+        //When
+        JqlQuery jqlQuery = QueryBuilder.byClass(Claim.class).build();
+        Changes changes = javers.findChanges(jqlQuery);
+
+        //Then
+        assertThat(changes.size()).isEqualTo(1);
+        assertThat(changes.get(0).toString()).isEqualTo("ValueChange{ 'claimStatus' value changed from 'ACTIVE' to 'PENDING' }");
+    }
```

Comment:
What happend is an address is modified, or a claim is updated without calling ClaimRepository.save?

Reply:
We don't have an address repository to update an address on it's own and we don't have any other way of updating a claim. The claim repository is the own way of updating a claim, so just added the javers annotation there.